code/datums/mind.dm

```diff
@@ -98,16 +98,29 @@
 	SSticker.minds -= src
 	QDEL_LIST(antag_datums)
 	QDEL_NULL(language_holder)
-	current = null
+	set_current(null)
 	return ..()
 
+/datum/mind/proc/set_current(mob/new_current)
+	if(new_current && QDELETED(new_current))
```

Comment:
isn't there a define that just checks gc_destroyed.

Reply:
yes, but it'd runtime if passed a null